test/old/testdir/gen_opt_test.vim

```diff
@@ -139,6 +139,7 @@ let test_values = {
       \ 'wildcharm': [[-1, 0, 100, 'x', '^Y', '^@', '<Esc>', '<', '^'],
       \		['', 'xxx', '<xxx>', '<t_xxx>', '<Esc', '<t_xx', '<C-C>',
       \		'<NL>', '<CR>', K_KENTER]],
+      \ 'winborder': [['rounded', 'none', 'single', 'solid'], ['xxx']],
```

Comment:
Still wrong place. It should be in the `Nvim-only options` section.

Reply:
Ops I searched wrong and overlooked it. now should be right.